src/platform/icelake/include/platform/alh.h

```diff
@@ -0,0 +1,84 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2019 Intel Corporation. All rights reserved.
+//
+// Author: Slawomir Blauciak <slawomir.blauciak@linux.intel.com>
+
+#ifndef __PLATFORM_ALH_H__
+#define __PLATFORM_ALH_H__
+
+#include <stdint.h>
+
+/**
+ * \brief ALH Handshakes
+ * Stream ID -> DMA Handshake map
+ * -1 identifies invalid handshakes/streams
+ */
+const uint8_t alh_handshake_map[] = {
+	-1,	/* 0  */
+	-1,	/* 1  */
+	-1,	/* 2  */
+	-1,	/* 3  */
+	-1,	/* 4  */
+	-1,	/* 5  */
+	-1,	/* 6  */
+	22,	/* 7  */
```

Comment:
Could you include descriptions of used handshakes?
Or maybe even make them defines, so they would describe themselves.

Reply:
The firmware would never use those handshakes in the code directly, so the defines would remain unused. We just retrieve the right handshake by using the stream_id we get from the driver.

Reply:
OK, so maybe at least description in the comment?